dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/tracker/validation/TrackedEntityImportValidationTest.java

```diff
@@ -92,6 +92,28 @@ void successValidationWhenTrackedEntityAttributeHasValidOptionValue()
         assertNoErrors( importReport );
     }
 
+    @Test
+    void failValidationWhenTrackedEntityAttributeHasWrongMultiTextValue()
```

Comment:
I think this duplicates tests unit tests in dhis-2/dhis-services/dhis-service-tracker/src/test/java/org/hisp/dhis/tracker/validation/validator/trackedentity/AttributeValidatorTest.java

I would therefore remove this one in favor of the unit tests

Reply:
ok, removed.